### DISCUSSION TITLE
Disable auto-break long lines and auto-add alignment code

### DISCUSSION THREAD

**Discussion by mongeroid** [SubtitleEdit/subtitleedit#5360] [Q&A]
Hello everyone, 

I recently started using Subtitle edit again after a long time. I'm having trouble because the program seems to make modifications automatically. Specifically, the program is breaking long lines once they go over the max number of characters, creating a third line despite having the max number of lines set at 2. The program is also adding {/an1} tags throughout randomly. Subtitle Edit doesn't do this in real-time, it only appears in the saved file, not in what I'm seeing. To see it in Subtitle Edit I would have to close and reopen the file.

I have already disabled auto breaking long lines in fix common errors, but it still happens.

I know I can increase the max number of characters, but the ones I didn't correct myself were only over by 1. Any help in these matters would be greatly appreciated.

**Comment by niksedk** [comment]
The SCC format has a max limit of 32 chars per line.

I'll try to improve the {\an1}...

**Reply by mongeroid** [reply]
Thanks a lot! Total oversight, getting used to time stamping again. 

Thanks for looking into the {\an1}!
